src/config.json

```diff
@@ -1258,6 +1258,70 @@
   ],
   "blacklist": [
     "xn--opensablog-vmb.com",
+    "airdrop.lacoste.cat",
+    "lacoste.cat",
+    "a156metamask.com",
+    "app-wallet-coinbase.googleystore.com",
+    "appuniswap.org",
+    "aptos-pancakeswap.finance",
+    "autholedgwallaps.org",
+    "authorise-coinbase.com",
+    "binance-exchange.com",
+    "binancewallet.org",
+    "coinbase-account49.dynamic-dns.net",
+    "coinbase-login-auth32.mefound.com",
+    "coinbase-secure-account.com",
+    "coinbase-sign-auth.dynamic-dns.net",
+    "coinbase.com---crypto.space",
+    "coinbaseverify.live",
+    "coinbasewallet.googleystore.com",
+    "dappsverificaion.work.gd",
+    "defiethereum.com",
+    "degods.trade",
+    "dgerguard.co",
+    "le.dgerguard.com",
+    "ethereum-connect.com",
+    "ethereum-defiaccess.com",
+    "ethereum-deficonnect.com",
+    "free-ethereum.tk",
+    "kyc-metamask-i0applicationoffceveriflcation10.neconme.com",
+    "login-kraken2022.com",
+    "login.coinbase.accounts.h7th.us",
+    "login.coinbase.accounts.ohveegitu1r.app",
+    "login.uniswap-global.network",
+    "loginuniswap.org",
+    "metamask-about.pages.dev",
+    "metamask-metamask.io",
+    "metamask-online.com",
+    "metamask.run",
+    "metamaskdown.co",
+    "opensea.nom.co",
+    "opensea.org.kz",
+    "opensea.org.pl",
+    "openseacryptomining.us",
+    "pancakesvvap.com",
+    "pancakeswap-page.net",
+    "pancakeswap.com.cm",
+    "pancakeswap.eu.com",
+    "pancakeswap.org.mx",
+    "pancakeswap.uk.net",
+    "pancakeswwp.com",
+    "pankaceswap.org.kz",
+    "password-update-coinbase.com",
+    "resetbridgedao.com",
+    "rubic-exchange-claims-rewards-erc20-token.com",
+    "securedeskmetamask.com",
+    "supaknottss.netlify.app",
+    "supportprocoinbase.com",
+    "truspad-pancakeswap.com",
+    "uniethereum.com",
+    "uniswapapes.com",
+    "uniswape.com.aviaryhotel.com",
```

Comment:
have the owners of this domain been contacted? looks like a dns hijack.

Reply:
I emailed them, no reply yet.

A case of ATO most likely.
![image](https://user-images.githubusercontent.com/49607867/199197796-a736775f-8545-4ec6-8220-ddc7fb145b9a.png)

However - Pretending to be "hacked" is a bad tactic often employed by threat actors.
It extends their lifespan and sometimes saves the domain for recycling.

Often there are just façade /placeholder domains, not representing a legitimate entity site whatsoever and just imaginary businesses.

Then wqhen the registrar or hosting asks them about the abuse they are like "Ruh Roh we were hacked"

See here
https://twitter.com/dubstard/status/1579758390334853121

Reply:
and here https://twitter.com/dubstard/status/1572291916548698112